lumicks/pylake/kymo.py

```diff
@@ -4,6 +4,8 @@
 
 import numpy as np
 
+from lumicks.pylake.detail.bead_cropping import find_beads
+
```

Comment:
We use relative imports for `pylake` stuff (absolute imports in the tests). I think it'll also get rid of the extra space

```suggestion
from .detail.bead_cropping import find_beads
```